src/index.ts

```diff
@@ -0,0 +1,52 @@
+import * as Sentry from '@sentry/node';
+import dotenv from 'dotenv';
+import yargsInit from 'yargs';
+
+import {renderServer} from './renderServer';
+import {renderStream} from './renderStream';
+import {validateStyleConfig} from './validate';
+
+dotenv.config();
+Sentry.init({dsn: process.env.SENTRY_DSN});
+
+/**
+ * Load external style configurations
+ */
+function styleConfigLoader(path: string) {
+  const config = require(/* webpackIgnore: true */ path).default;
+
+  if (!validateStyleConfig(config)) {
+    throw new Error('Invalid style configuration');
+  }
+  return config;
+}
+
+yargsInit(process.argv.slice(2))
```

Comment:
Would it make sense to have this in like `src/cli/` or something?

Reply:
oh never mind, I see what it's doing now

Reply:
`yargs` is basically the parser / runner.

The whole thing is just one big cli tool really